src/test/end-to-end/pages/amendCaseDetails/amendCaseDetailsWithCasePosition.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const commonConfig = require('../../data/commonConfig.json');
+
+module.exports = async function (clerkResponcible, casePosition, physicalLocation, conciliationTrack) {
+
+    const I = this;
+    await I.selectOption('#clerkResponsible', clerkResponcible);
+    await I.selectOption('#positionType', casePosition);
+    await I.selectOption('#fileLocation', physicalLocation);
+    await I.selectOption('#conciliationTrack', conciliationTrack);
+    await I.navByClick(commonConfig.continue);
+    await I.navByClick(commonConfig.continue);
+    await I.click(commonConfig.submit);
+    await I.see(commonConfig.moveAcceptedCaseToCloseCaseErrorCheck);
```

Comment:
Should this be in commonConfig or Constants.js

Reply:
Will take care that code improvements in next pr